build-status.el

```diff
@@ -375,8 +375,7 @@ Signals an error if the response does not contain an HTTP 200 status code."
 ;;;###autoload
 (define-minor-mode build-status-mode
   "Monitor the build status of the buffer's project."
-  :global t
```

Comment:
I bisect my custom mode-line fragments, found another error in fragment `*buffer-info` caused error `stringp nil`. This cause my custom mode-line failed. Thanks.